src/grid/Row.js

```diff
@@ -1,18 +1,26 @@
 import React, { PropTypes } from 'react';
-import { View, TouchableOpacity } from 'react-native';
+import { View, StyleSheet, TouchableOpacity } from 'react-native';
 
 const Row = (props) => {
-  const {style, size, onPress, activeOpacity} = props;
+  const {containerStyle, size, onPress, activeOpacity} = props;
 
-  const styles = {
-    flex: (size) ? size : (style && style.height) ? 0 : 1,
-    flexDirection: 'row',
-    ...style,
-  };
+  const styles = StyleSheet.create({
```

Comment:
@Monte9 The reason to have styles inside the `Row` object is because we need access to the props `containerStyle` and `containerStyle.height`